changes/9338.ramp_fitting.rst

```diff
@@ -0,0 +1 @@
+Restore __all__ definition that was removed in an earlier PR
```

Comment:
If user won't notice anything, not sure if this actually needs a change log. I'll leave that up to you. If you want to keep it, here is a suggestion.
```suggestion
Restore ``__all__`` definition that was removed in an earlier PR.
```

Reply:
Thanks, I kept it with your suggestion!